src/form/radiology-orders/radiology-order-basket-panel/radiology-order-basket-panel.scss

```
@@ -3,16 +3,16 @@
 @import '@openmrs/esm-styleguide/src/vars';
 
 .desktopTile {
-  border-left: 4px solid #6929c4;
+  border-left: 4px solid #18cd69;
```

```suggestion
  border-left: 4px solid colors.$color-o1;
```
We should use carbon color tokens